server/conditions/conditions.go

```diff
@@ -0,0 +1,150 @@
+		202	Thunderstorm	thunderstorm with heavy rain
+		210	Thunderstorm	light thunderstorm
+		211	Thunderstorm	thunderstorm
+		212	Thunderstorm	heavy thunderstorm
+		221	Thunderstorm	ragged thunderstorm
+		230	Thunderstorm	thunderstorm with light drizzle
+		231	Thunderstorm	thunderstorm with drizzle
+		232	Thunderstorm	thunderstorm with heavy drizzle
+		300	Drizzle		light intensity drizzle
+		301	Drizzle		drizzle
+		302	Drizzle		heavy intensity drizzle
+		310	Drizzle		light intensity drizzle rain
+		311	Drizzle		drizzle rain
+		312	Drizzle		heavy intensity drizzle rain
+		313	Drizzle		shower rain and drizzle
+		314	Drizzle		heavy shower rain and drizzle
+		321	Drizzle		shower drizzle
+		500	Rain		light rain
+		501	Rain		moderate rain
+		502	Rain		heavy intensity rain
+		503	Rain		very heavy rain
+		504	Rain		extreme rain
+		511	Rain		freezing rain
+		520	Rain		light intensity shower rain
+		521	Rain		shower rain
+		522	Rain		heavy intensity shower rain
+		531	Rain		ragged shower rain
+		600	Snow		light snow
+		601	Snow		Snow
+		602	Snow		Heavy snow
+		611	Snow		Sleet
+		612	Snow		Light shower sleet
+		613	Snow		Shower sleet
+		615	Snow		Light rain and snow
+		616	Snow		Rain and snow
+		620	Snow		Light shower snow
+		621	Snow		Shower snow
+		622	Snow		Heavy shower snow
+		701	Mist		mist
+		711	Smoke		Smoke
+		721	Haze		Haze
+		731	Dust		sand/ dust whirls
+		741	Fog			fog
+		751	Sand		sand
+		761	Dust		dust
+		762	Ash			volcanic ash
+		771	Squall		squalls
+		781	Tornado		tornado
+		800	Clear		clear sky
+		801	Clouds		few clouds: 11-25%
+		802	Clouds		scattered clouds: 25-50%
+		803	Clouds		broken clouds: 51-84%
+		804	Clouds		overcast clouds: 85-100%
+	*/
+	switch i {
+	case 200, 201, 202, 210, 211, 212, 221, 230, 231, 232, 531:
+		return "thunderstorm"
+	case 300, 301, 302:
+		return "drizzle"
+	case 310, 311, 312, 313, 314, 321, 500, 520, 521, 522:
+		return "rain_light"
+	case 501:
+		return "rain"
+	case 502, 503, 504:
+		return "rain_heavy"
+	case 511:
+		return "freezing_rain"
+	case 600, 620:
+		return "snow_light"
+	case 601, 621:
+		return "snow"
+	case 602, 622, 615, 616:
+		return "snow_heavy"
+	case 611:
+		return "sleet_light"
+	case 612:
+		return "sleet"
+	case 613:
+		return "sleet_heavy"
+	case 701, 711, 721, 731, 751, 761, 762, 771:
+		return "fog_light"
+	case 741:
+		return "fog"
+	case 781:
+		return "tornado"
+	case 800:
+		return "clear_" + dayOrNight
+	case 801:
+		return "mostly_clear_" + dayOrNight
+	case 802:
+		return "partly_cloudy_" + dayOrNight
+	case 803:
+		return "mostly_cloudy"
+	case 804:
+		return "cloudy"
+	default:
+		return "unknown"
+	}
+}
+
+// MoonPhase returns friendly name of the icon of  the moon phase
+// when provided the status
+func MoonPhase(i float64) string {
+	/*
+		Moon phase.
+		0 and 1 are 'new moon',  waxing crescent
+		0.25 is 'first quarter moon',  waxing gibbous
+		0.5 is 'full moon' and  waning gibbous
+		0.75 is 'last quarter moon'  'waning crescent'
```

Comment:
```suggestion
		0 and 1:    'new moon',  
		0 < 0.25:   'waxing crescent'
		0.25:       'first quarter moon'
		0.25 < 0.5: 'waxing gibbous'
		0.5:        'full moon'
		0.5 < 0.75: 'waning gibbous'
		0.75:       'last quarter moon'
		0.75 < 1:   'waning crescent'
```

This is not really important, but this comment is incomplete.